strings/british_talk.json

```diff
@@ -0,0 +1,118 @@
+{
+
+    "words": {
+        "id": "BBM Pin",
```

Comment:
```suggestion
        "id": "BBM Pin",
        "ass": "batty",
        "thing": "ting",
        "crazy": "mad",
        "cool": "calm",
        "awesome": "lit",
        "many":"bare",
        "fries": "chips",
        "sidewalk": "pave",
        "australia": "penal colony",
```

Reply:
Thing is already in the list. Also shouldn't it be pavement?